bin/timeout_ssh

```diff
@@ -51,9 +52,15 @@ def main(argv):
     else:
         timeout_command = 'timeout'
 
+    # the ssh arg is required for become-pass because
+    # ansible checks for an exact string match of ssh
+    # anywhere in the command array
+    # See https://github.com/ansible/ansible/blob/stable-2.3/lib/ansible/plugins/connection/ssh.py#L490-L500
+    # the code below will remove the ssh argument before running the command
     executable_args = [arg for arg in argv[1:]
                        if (not arg.startswith(ARG_EXECUTABLE) and
-                           not arg.startswith(ARG_TIMEOUT))]
+                           not arg.startswith(ARG_TIMEOUT) and
+                           arg != 'ssh')]
```

Comment:
Nit: maybe declare a global variable `SSH = 'ssh'` and then use it here?

Reply:
👍